modules/sand/src/main/ts/ephox/sand/api/SandHTMLElement.ts

```diff
@@ -1,6 +1,8 @@
-import { Resolve } from '@ephox/katamari';
+import { Resolve, Type } from '@ephox/katamari';
 import * as Global from '../util/Global';
 
+const getPrototypeOf = Object.getPrototypeOf;
```

Comment:
I stored this to ensure the API isn't messed with by polyfills. We currently do the same thing in other places for this reason, e.g. `Obj`.